kartothek/io_components/write.py

```diff
@@ -55,7 +55,7 @@ def persist_common_metadata(partition_list, update_dataset, store, dataset_uuid)
         for tab, tm in mp.table_meta.items():
             tm_dct[tab].add(tm)
 
-    if update_dataset:
+    if update_dataset and update_dataset.exists:
```

Comment:
test?

Reply:
Again no dedicated unit test but its used all over the place. What test do you have in mind?